examples/input/gamepad_stick_input.rs

```diff
@@ -0,0 +1,135 @@
+use bevy::{input::gamepad::GamepadSettings, prelude::*};
+
+const WINDOW_SIZE: f32 = 300.0;
+const CROSSHAIR_SIZE: f32 = 16.0;
+const FONT: &str = "fonts/FiraMono-Medium.ttf";
+const FONT_SIZE: f32 = 18.0;
+const LIVEZONE_COLOR: Color = Color::GRAY;
+const DEADZONE_COLOR: Color = Color::rgb(0.4, 0.4, 0.4);
+
+#[derive(Component)]
+struct Crosshair;
+
+#[derive(Component)]
+struct CoordinateText;
+
+fn main() {
+    App::new()
+        .insert_resource(WindowDescriptor {
+            title: "Gamepad Stick Input".to_owned(),
+            width: WINDOW_SIZE,
+            height: WINDOW_SIZE,
+            ..Default::default()
+        })
+        .insert_resource(ClearColor(DEADZONE_COLOR))
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(update_position)
+        .run();
+}
+
+fn update_position(
+    mut xform_query: Query<&mut Transform, With<Crosshair>>,
```

Comment:
For consistency with other examples, I think `crosshair_query` would be better here.